_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -636,15 +624,14 @@ use google to find a list of pch symbols that you can use in `R`.
 ``` {r challenge-code-plot-color, results="hide", warning= FALSE, echo=FALSE}
 ## 1
 # Read the shapefile file
-State.Boundary.US <- readOGR("NEON-DS-Site-Layout-Files/US-Boundary-Layers",
-          "US-State-Boundaries-Census-2014")
+State.Boundary.US <- st_read("data/NEON-DS-Site-Layout-Files/US-Boundary-Layers/US-State-Boundaries-Census-2014.shp")
```

Comment:
We have inconsistent naming here. let's avoid periods given they can be methods in other languages and sometimes in r... i'm happy to help or just rename this to 

state_boundary_us 
just a quick find and replace

i've noticing lots of naming inconsistencies that were already there!

Reply:
lets leave this for a future PR